src/processorsFiles/models/database/cassandra/cassandraConnector.js

```diff
@@ -0,0 +1,42 @@
+let logger = require('../../../../common/logger');
+let cassandra = require('cassandra-driver');
+let client = {};
+
+const GET_FILE = 'SELECT file FROM processorsFiles';
```

Comment:
rename to `processors`

Reply:
I have dropped the cassandraConnector.js since I have to completely with the implementation over Sequelize, before moving on to Cassandra

Reply:
sounds good